src/mjolnir/pbfgraphparser.cc

```diff
@@ -1479,8 +1479,7 @@ struct graph_callback : public OSMPBF::Callback {
     }
 
     // Infer cul-de-sac if a road edge is a loop and is low classification.
-    if (!way_.roundabout() && loop_nodes_.size() != nodes.size() &&
-        (way_.use() == Use::kRoad || way_.use() == Use::kServiceRoad) &&
+    if (!way_.roundabout() && loop_nodes_.size() != nodes.size() && way_.use() == Use::kRoad &&
```

Comment:
this was a data bug that marks drive throughs as cul-de-sacs, which can cause dive bombs

![Screenshot from 2021-03-17 11-39-35](https://user-images.githubusercontent.com/2676277/111495457-c428c000-8715-11eb-86a8-e8e0b528a663.png)

Reply:
nice